lib/utils/eventuallyUpdate.js

```diff
@@ -6,6 +6,7 @@ const path = require('path');
 const os = require('os');
 const stream = require('stream');
 const { promisify } = require('util');
+const fs = require('fs');
```

Comment:
Let's require `fs.promises` as `fsp`:

```
const fsp = require('fs').promises;
```

Reply:
sure. makes sense

Reply:
Thanks, please request a review from me when it's ready for reviewing again :bow: